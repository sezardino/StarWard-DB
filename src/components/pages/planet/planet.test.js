import React from 'react';
import {shallow} from 'enzyme';

import PlanetPage from './planet';

describe('Component PlanetPage', () => {
	const mockMatch = {params: {id: 'Test id'}};
	const component = shallow(<PlanetPage match={mockMatch} />);
	it('Component rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
