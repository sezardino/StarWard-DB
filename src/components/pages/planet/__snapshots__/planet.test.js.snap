// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component PlanetPage Component rendered correctly 1`] = `"<Row left={{...}} right={{...}} />"`;
